src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/takecard/TakeCard.kt

```diff
@@ -56,18 +77,18 @@ class TakeCard(val take: Take, player: IAudioPlayer, takeEventObservable: Observ
     }
 
     private fun firePauseEvent() {
-        fireEvent(TakeEvent(TakeEvent.PAUSE))
+        fireEvent(PauseEvent())
     }
 }
 
 private fun createTakeCard(
     take: Take,
     player: IAudioPlayer,
-    takeEventObservable: Observable<TakeEvent?>,
+    playOrPauseEventObservable: Observable<PlayOrPauseEvent?>,
```

Comment:
I thought we had an issue putting nullables in observables?

Reply:
Oh, yeah, this is easy to forget (and you don't find out until a null is observed at runtime).